src/main/java/org/springframework/data/release/infra/LicenseHeaderCommands.java

```diff
@@ -108,27 +109,46 @@ public void updateLicenseHeaders(@CliOption(key = "", mandatory = true) TrainIte
 		});
 	}
 
-	private int updateLicense(int year, ModuleIteration module) {
+	private int updateLicense(String year, ModuleIteration module) {
 
-		return replaceInFiles(module.getSupportedProject(), (file, content) -> {
+		return replaceInFiles(module.getSupportedProject(), (file, content) ->
+				this.updateLicenseHeaderInFile(file, content, year, module.getSupportStatus()));
+	}
 
-			String contentToUse = content;
+	String updateLicenseHeaderInFile(File file, String content, String year, SupportStatus supportStatus) {
```

Comment:
Looks like the method could be static. Also, I wonder whether we could/should introduce an enum to distinguish between file types. Right now, this happens inside the method and handing in a `File` and its content made me already wonder what we do with the file here.

Reply:
Made method static and introduced the `FileType` enum.